site/documentation/content/concepts/device-identity.md

```diff
@@ -147,3 +147,34 @@ After having verified the client certificate using the trust anchor(s), the prot
 the client certificate's *subject DN* and invokes the Credentials API's
 [*get Credentials*]({{< relref "/api/credentials#get-credentials" >}}) operation in order to retrieve the
 [*x509-cert* type credentials]({{< relref "/api/credentials#x509-certificate" >}}) that are on record for the device.
+
+### JSON Web Token based Authentication
+
+The MQTT protocol adapter supports authentication of devices with a JSON Web Token (JWT) based mechanism. In this case,
+the protocol adapter verifies the JWT by using the public key / certificate that is on record for the device in the
+device registry. Further more does the protocol adapter check if the JWT is valid. To be valid the JWT has to provide
+the following information in its header and payload.
+
+The JWT header has to provide two fields that indicate the type of token (`typ`) and the signing algorithm (`alg`).
+Both fields are mandatory and the type has to be `JWT`. For the signing algorithm `RS256` and `ES256` are supported.
+The algorithm specified in the header must match at least one of the
+[*asymmetric-key* type credentials]({{< relref "/api/credentials#asymmetric-key" >}}) registered for the device.
+
+The JWT payload must at least contain the claims `iat` ("issued at") and `exp` ("expiration time") with values provided
+in [Unix time](https://en.wikipedia.org/wiki/Unix_time). The `iat` claim marks the timestamp the token was created at
+and will also represent the start of its validity period. The `nbf` ("not before") claim is therefore not required and
+will be ignored. The `exp` claim signifies the point in time after which the token becomes invalid.
```

Comment:
It feels strange to support authentication of devices using a JWT and then not have the JWT contain `sub` and `iss` (and `aud`) claims, binding the signature to the device's identity.
Is there a particular reason why you want to omit them?